README.md

```diff
@@ -509,9 +504,6 @@ used, but the full path should be in your own copy of the file):
     have been issues seen using the CIAO binaries on certain OS X
     systems.
 
-In all cases, the same version of `gfortran` should be used to build
-Sherpa and XSPEC, in order to avoid possible incompatibilities.
-
 If there are problems building, or using, the module, then the other
```

Comment:
We have this paragraph which talks about `gfortran_xxx` settings. This seems at odds with our brave, new, Fortran-Free world. I think I added this paragraph, but I don't know of a situation where we've had to change these settings (which is not a solid proof that they are not needed, I admit).

I think we should leave this paragraph in for now, but when we get to the XSPEC 12.10.0 PRs we can look at whether this paragraph still has any use.

Reply:
I'll clarify that part. Gfortran is not needed to build the xspec extension. Only libgfortran is, and it needs to be the same (or same version) as the one used to build xspec.

Reply:
Maybe it would help to include a setup.cfg.xspec with some of the options preset to a common scenario, with placeholders for the actual paths the user will have to change.